src/components/AppLayout/Header/components/ProviderDetails/HidePairingModule.tsx

```diff
@@ -0,0 +1,6 @@
+import 'src/components/AppLayout/Header/components/ProviderDetails/hidePairingModule.css'
+import { ReactElement } from 'react'
+
+const HidePairingModule = (): ReactElement => <></>
```

Comment:
You could return null here to spare mounting a component in the tree, albeit just a fragment.